test/test_gpu_gather_ops.cpp

```diff
@@ -415,4 +443,47 @@ TEST_F(NVFuserTest, FusionTorchGatherIndexTvExtentIsOne_CUDA) {
       &fusion, cg_outputs, aten_inputs, {tv_out_ref}, __LINE__, __FILE__);
 }
 
+// Test take_along_axis with a broadcast index tensor
+TEST_F(NVFuserTest, TakeAlongBroadcastIndex_CUDA) {
+  for (const auto index_dim : {1, 3}) {
+    auto fusion_ptr = std::make_unique<Fusion>();
+    Fusion& fusion = *fusion_ptr.get();
+    FusionGuard fg(&fusion);
+
+    auto tv0 = makeSymbolicTensor(3);
```

Comment:
Should we also test the case where tv0 is also a broadcast tensor? I think this is supported by numpy.

Reply:
As mentioned above, added a test, `TakeAlongBroadcastInput_CUDA`, but it doesn't work right now.